docs/buildah-pull.md

```diff
@@ -42,6 +42,10 @@ The image ID of the image that was pulled.  On error 1 is returned.
 
 ## OPTIONS
 
+**--all-tags, a**
+
+All tagged images in the repository will be downloaded.
```

Comment:
I'd suggest "pulled" over "downloaded".  I know the Docker dock uses downloaded, but think it should be pulled there too as we never talk about download prior to this.